nilearn/connectivity/matrix_operations.py

```diff
@@ -0,0 +1,352 @@
+import warnings
```

Comment:
Can we change "matrix_operations" to "connectome_matrices" since its the base to use for learning any functional connectomes ?